src/types/routerTypes.tsx

```diff
@@ -52,6 +52,7 @@ export interface RouteParamList {
 
   // Tabs
   homeTab: {}
+  walletsTab: {}
```

Comment:
Rename to `assetsTab` so the tabs align with the titles

Reply:
Since it's optional, I'll skip to save time